command-line/command-line-advanced.md

```diff
@@ -3,10 +3,14 @@
 ### Projected Time
 
 About 1 hour
+- 20 minutes for Lesson
+- 20 minutes for Guided Practice
+- 15 minutes for Independent Practice
+- 10 minutes for Check for Understanding
 
 ### Prerequisites
 
-[Basic command line class](command-line-interface.md).
+[Basic command line](command-line-interface.md).
```

Comment:
(../command-line-interface.md)

Reply:
If I change to this link then it says error 404 not found. @Kundan28 Please help to solve this issue

Reply:
(command-line-interface.md) is actually correct, since it is in the same folder as the outline.